server/src/main/java/org/elasticsearch/search/fetch/QueryFetchSearchResult.java

```diff
@@ -24,6 +24,7 @@ public final class QueryFetchSearchResult extends SearchPhaseResult {
 
     public QueryFetchSearchResult(StreamInput in) throws IOException {
         super(in);
+        // TODO: Delegate refcounting to QuerySearchResult
```

Comment:
Is it worth adding the other PR number this references?

Reply:
yeah, that's a good idea.  I'll add it in.